app/views/apps/_app_list.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="panel panel-default">
+  <div class="panel-heading"><%= app_list.title %></div>
+  <div class="list-group">
+
+    <% OodAppGroup.groups_for(apps: app_list.apps, group_by: :subcategory).each_with_index do |g, index| %>
+      <%= content_tag "p", g.title, class: "list-group-item header" unless g.title.empty? %>
+      <%= render partial: "apps/app_list_item", collection: g.apps, as: :app, locals: { current_app_token: local_assigns[:current_app_token] }%>
```

Comment:
Do we need to prepend "apps/" to the partial path? We didn't do that below for this partial.

Reply:
I believe you can remove the `locals: { ... }` on this line as it isn't used in the corresponding partial.